packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue

```diff
@@ -136,68 +140,145 @@ const handlePressDownKey = () => {
          :class="{
-             [styleType] : true,
-             invalid,
-             disabled,
-             'read-only': readOnly,
-             active: state.proxyVisibleMenu && !readOnly,
-             [size] : true,
-             'is-fixed-width': isFixedWidth,
+             [props.styleType] : true,
+             [props.appearanceType]: true,
+             invalid: props.invalid,
+             disabled: props.disabled,
+             active: state.proxyVisibleMenu && !props.readOnly,
+             'read-only': props.readOnly,
+             'is-fixed-width': props.isFixedWidth,
+             'highlight-selection-state': props.highlightSelectionState,
+             'selected': state.selectedItems.length > 0,
          }"
     >
-        <p-icon-button v-if="styleType === SELECT_DROPDOWN_STYLE_TYPE.ICON_BUTTON"
-                       ref="targetRef"
-                       :name="buttonIcon || (state.proxyVisibleMenu ? 'ic_chevron-up' : 'ic_chevron-down')"
-                       :activated="state.proxyVisibleMenu"
-                       :disabled="disabled"
-                       color="inherit"
-                       class="icon-button"
-                       @click="handleClick"
-                       @keydown.down="handlePressDownKey"
+        <p-i v-if="props.styleType === SELECT_DROPDOWN_STYLE_TYPE.ICON_BUTTON"
+             ref="targetRef"
+             :name="props.buttonIcon || (state.proxyVisibleMenu ? 'ic_chevron-up' : 'ic_chevron-down')"
+             :activated="state.proxyVisibleMenu"
+             :disabled="props.disabled"
+             color="inherit"
+             class="only-icon-button"
+             style-type="tertiary"
+             @click="handleClick"
+             @keydown.down="handlePressDownKey"
         />
         <button v-else
                 ref="targetRef"
                 class="dropdown-button"
-                :class="{'text-only': (styleType === SELECT_DROPDOWN_STYLE_TYPE.TRANSPARENT && readOnly)}"
+                :class="{'text-only': (props.styleType === SELECT_DROPDOWN_STYLE_TYPE.TRANSPARENT && props.readOnly)}"
                 @click="handleClick"
                 @keydown.down="handlePressDownKey"
         >
             <span class="text"
-                  :class="{placeholder: !state.selectedItem}"
+                  :class="{placeholder: state.selectedItems.length === 0}"
             >
                 <slot name="default"
-                      v-bind="{item: state.selectedItem}"
+                      v-bind="{item: state.selectedItems}"
                 >
-                    {{
-                        state.selectedItem ?
-                            (state.selectedItem.label || state.selectedItem.name || '') :
-                            (placeholder || $t('COMPONENT.SELECT_DROPDOWN.SELECT'))
-                    }}
+                    <strong v-if="props.innerLabel !== undefined && state.selectedItems.length > 0">
+                        {{ props.innerLabel || 'Label' }}:
```

Comment:
question: Is there no need to translate "Label"?

Reply:
I've checked where the translation is used in the console and decided to proceed if necessary!